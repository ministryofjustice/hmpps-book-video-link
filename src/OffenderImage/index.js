import OffenderImage from './OffenderImage'

export default OffenderImage
