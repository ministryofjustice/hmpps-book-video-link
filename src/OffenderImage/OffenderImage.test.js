import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { BREAKPOINTS } from '@govuk-react/constants'
import { StyledImage } from './OffenderImage'

test('renders with the correct styles', () => {
  const tree = renderer.create(<StyledImage />).toJSON()
  expect(tree).toMatchSnapshot()
  expect(tree).toHaveStyleRule('max-width', '70px')
  expect(tree).toHaveStyleRule('max-width', '90px', { media: `(min-width: ${BREAKPOINTS.DESKTOP})` })
})
