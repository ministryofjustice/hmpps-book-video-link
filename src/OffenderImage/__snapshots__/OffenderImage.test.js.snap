// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with the correct styles 1`] = `
.c0 {
  max-width: 70px;
}

@media (min-width:769px) {
  .c0 {
    max-width: 90px;
  }
}

<img
  className="c0"
/>
`;
